### PR TITLE
Update ncco examples

### PR DESCRIPTION
**Pull request by kellyjandrews**
## Description

Updated ncco examples to use the ncco-example repository.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: voice/asr.js [hunk 1]
```diff
@@ -4,20 +4,23 @@ const bodyParser = require('body-parser')
...
-      eventUrl: [`${request.protocol}://${request.get('host')}/webhooks/asr`],
       speech: {
-        endOnSilence: 1,
-        language: "en-US",
-        uuid: [request.query.uuid]
-      }
-    }
-  ]
+        uuid: ['aaaaaaaa-bbbb-cccc-dddd-0123456789ab'],
+        language: 'en-gb',
```
Comment (hummusonrails) [inline voice/asr.js:19]: Why did we switch to `en-GB` away from `en-US`?

File: voice/connect-an-inbound-call.js [hunk 2]
```diff
@@ -1,18 +1,26 @@
 require('dotenv').config({path: __dirname + '/../.env'});
-const YOUR_SECOND_NUMBER = process.env.YOUR_SECOND_NUMBER;
+const TO_NUMBER = process.env.YOUR_SECOND_NUMBER;
```
Comment (hummusonrails) [inline voice/connect-an-inbound-call.js:2]: Should the env variable match the constant naming -- i.e. `TO_NUMBER` and `TO_NUMBER`?

File: voice/dtmf.js [hunk 3]
```diff
@@ -7,13 +7,14 @@ const onInboundCall = (request, response) => {
   const ncco = [
     {
       action: 'talk',
-      text: 'Please enter a digit'
+      text:
+        "Hi, this call collects dtmf input using Vonage's Voice API. Please press a couple of buttons on your phone keypad",
+      voice_name: 'Amy',
     },
     {
       action: 'input',
```
Comment (hummusonrails) [inline voice/dtmf.js:15]: I think there needs to be more here per the changes to the `input` NCCO. See the example in the docs https://developer.nexmo.com/voice/voice-api/ncco-reference#input

File: voice/ivr-menu.js [hunk 4]
```diff
@@ -4,16 +4,16 @@ const bodyParser = require('body-parser')
...
+      text:
+        "Hi, this call collects dtmf input using the Vonage Voice API. Please press a couple of buttons on your phone keypad",
+      voice_name: 'Amy',
     },
     {
       action: 'input',
-      maxDigits: 1,
-      eventUrl: [`${req.protocol}://${req.get('host')}/webhooks/dtmf`]
-    }
+    },
```
Comment (hummusonrails) [inline voice/ivr-menu.js:16]: Same as above, you need the `type: [ 'dtmf']`, etc from the `input` NCCO guide: https://developer.nexmo.com/voice/voice-api/ncco-reference#input

File: voice/record-a-message.js [hunk 5]
```diff
@@ -4,23 +4,16 @@ const bodyParser = require('body-parser')
...
-        `${request.protocol}://${request.get('host')}/webhooks/recordings`
-      ]
     },
     {
       action: 'talk',
-      text: 'Thank you for your message. Goodbye.'
-    }
+      text:
+        'Hi, this call records this message. Check the parameters sent to the event url webhook endpoint and find the link you need to download this recording.',
+      voice_name: 'Amy',
```
Comment (hummusonrails) [inline voice/record-a-message.js:15]: This should be switched to `language`, the `voice_name` param is deprecated

File: voice/asr.js (hunk 1 above)
Comment (kellyjandrews) [reply voice/asr.js:19]: https://github.com/nexmo-community/ncco-examples/blob/gh-pages/asr-input.json

File: voice/connect-an-inbound-call.js (hunk 2 above)
Comment (kellyjandrews) [reply voice/connect-an-inbound-call.js:2]: I don't think that is necessary.

File: voice/dtmf.js (hunk 3 above)
Comment (kellyjandrews) [reply voice/dtmf.js:15]: I used this one - does the ncco example repo need to then be updated? https://github.com/nexmo-community/ncco-examples/blob/gh-pages/dtmf-input.json

**Comment by kellyjandrews** [comment]
@bencgreenberg I'm closing this pull request. I was asked to migrate to use https://github.com/nexmo-community/ncco-examples but it seems these are not properly updated.

File: voice/asr.js (hunk 1 above)
Comment (hummusonrails) [reply voice/asr.js:19]: 👍

File: voice/connect-an-inbound-call.js (hunk 2 above)
Comment (hummusonrails) [reply voice/connect-an-inbound-call.js:2]: I like consistency, but it's your code

File: voice/dtmf.js (hunk 3 above)
Comment (hummusonrails) [reply voice/dtmf.js:15]: Indeed. I'll raise a PR to do so on that one, thanks!

**Pull request closed without merging**